README.md

```diff
@@ -121,6 +121,21 @@ animator.start();
 animationView.cancelAnimation();
 ```
 
+You can add a color filter to the whole animation, a specific layer, or specific content within a layer:
+```java
+final ColorFilter colorFilter = new PorterDuffColorFilter(Color.RED, PorterDuff.Mode.SRC_ATOP);
```

Comment:
I will make one, but I feel that if an android dev is dealing with drawables and color filters they should learn which xfer mode suits their needs. 

I can see how this could generate the interest the learn other modes though.